test-network-function/identifiers/identifiers.go

```diff
@@ -333,6 +338,20 @@ test case requires the Deployment of the debug daemonset.
 		BestPracticeReference: bestPracticeDocV1dot2URL + " Section 6.2",
 	},
 
+	TestICMPv4ConnectivityIdentifierMultus: {
+		Identifier: TestICMPv4ConnectivityIdentifierMultus,
+		Type:       normativeResult,
+		Remediation: `Ensure that the CNF is able to communicate via the Multus network(s).  In some rare cases,
+CNFs may require routing table changes in order to communicate over the Multus network(s).  In other cases, if the
+Container base image does not provide the "ip" or "ping" binaries, this test may not be applicable.  For instructions on
```

Comment:
Since we started using nsenter, the "ping" binary is no longer needed to be inside the container image. If I'm not wrong, nsenter uses the binaries from the host where the tnf is running. This is applicable for both networking TCs (default+multus).

Reply:
I think you're right, I'll fix that.

Reply:
So I was wrong. It uses the debug pod's context to issue the nsenter command. Thanks @jun-rh .